db/migrate/20160322205357_remove_host_id_from_switch.rb

```diff
@@ -0,0 +1,41 @@
+class RemoveHostIdFromSwitch < ActiveRecord::Migration[5.0]
+  class Host < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
+    has_many :host_switches,
+             :dependent   => :destroy,
+             :class_name  => 'RemoveHostIdFromSwitch::HostSwitch'
+    has_many :switches,
+             :through => :host_switches,
+             :class_name  => 'RemoveHostIdFromSwitch::Switch'
```

Comment:
Actually they fit in 1 line comfortably, so one-lined.  Just not sure if the elements need to be aligned between the 2 lines. E.g. does the :classname need to be aligned between 2 lines?
`has_many :host_switches, :dependent => :destroy, :class_name => 'RemoveHostIdFromSwitch::HostSwitch'`
`has_many :switches, :through => :host_switches, :class_name => 'RemoveHostIdFromSwitch::Switch'`